### PR TITLE
ticket 19 finished, tests created

### PR DESCRIPTION
**Pull request by rodionross**

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -497,3 +496,96 @@ describe("PATCH /api/comments/:comment_id", () => {
...
+        designer: "test",
+        category: "dexterity",
+      })
+      .expect(400)
+      .then(({ body, status }) => {
+        expect(status).toBe(400);
+        expect(body.msg).toBe("bad request");
+      });
+  });
+});
```
Comment (pkaodev) [inline __tests__/api.test.js:591]: how do we want to handle someone sending an empty review?
what if the review contains swear words, do we want to block those?
what if the user has already posted another review a few seconds ago?
Just some inspiration if you want to expand it.

**Review by pkaodev — APPROVED** [review]
looks good, i'd add the test for handling an empty review being posted.  don't worry about the other things I wrote, just ideas.
